bin/input_module_ck_module.py

```diff
@@ -284,48 +180,43 @@ def login(username, password):
     populated by a dictionary that (among other things) contains the
     user's AccessToken. If the authentication fails, an exception is raised
     (actually a HTTPError: Bad Request)."""
-
-    url = 'https://api.pencildata.com/token'
+    url = ENDPOINT + 'token'
     data = {'userId': username, 'password': password}
     head = {"Content-Type": "application/json"}
     res = requests.request("POST", url, data=json.dumps(data), headers=head)
-
     return res.json()
-    
+
+
 def make_hash(val):
     hash_object = hashlib.sha256()
     hash_object.update(val.encode('utf-8'))
     return hash_object.hexdigest()
 
+
 ### This function is to register hash to blockchain
 def register(login_data, hash, storage="pencil"):
     """Register a file (by its SHA-256 hash) in your Pencildata account.
     Warning: this method does not check if the file hash exists in the
     registers. It returns the asset id for the file.
-
     Arguments:
     login_data: Access Token
     storage: 'public' or 'private'. Whether to store the file entry in the
     public or in the private database at the PencilDATA server."""
-
     datajson = {}
-
     datajson["hash"] = hash
     datajson["storage"] = storage
-    url = "https://api.pencildata.com/register/"
-
+    url = ENDPOINT + "register/"
     head = {"Content-Type": "application/json",
             "Authorization": "Bearer {0}".format(login_data['data']['accessToken'])}  # Request HTTP headers
     res = requests.request("POST", url, data=json.dumps(datajson), headers=head)
-
     return res.json()
 
+
 ### This function is to verify hash against hash in blockchain
 def verify(login_data, asset_id, hash, storage="pencil"):
```

Comment:
Pass `endpoint=ENDPOINT` as final parameter. Move `pencil` string to a constant on top called `STORAGE`

Reply:
Do this for `login`, `verify` and `register`